src/legacy/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -458,6 +458,12 @@ export function getUiSettingDefaults() {
            away or update the query. When enabled, dashboard panels will load together when all of the data is loaded, and
            searches will not terminate.`,
       }),
+      deprecation: {
+        message: i18n.translate('kbn.advancedSettings.courier.batchSearchesTextDeprecation', {
+          defaultMessage: 'This setting is deprecated and will be removed in Kibana 8.0.',
+        }),
+        url: 'https://www.elastic.co/guide/en/kibana/current/advanced-options.html',
```

Comment:
Question, should this URL use the DocLinksService instead of hardcoding the entire url?

Reply:
Message text LGTM